src/Plug-ins/RemoteAppExecution/SR.cs

```diff
@@ -19,5 +19,9 @@ namespace Monai.Deploy.InformaticsGateway.PlugIns.RemoteAppExecution
     internal static class SR
```

Comment:
What does SR stand for?

Reply:
No idea! something Victor added there are 4 SR.cs files throughout the solution! all contain static strings